tensorflow/lite/micro/kernels/cmsis-nn/conv.cc

```diff
@@ -13,10 +13,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+#include "arm_nnfunctions.h"
```

Comment:
@advaitjain CMSIS-NN provides arm_nnfunctions.h as the sole interface API file to its users. For the glue functions here, I would say that it is just the API's that are of interest than how the arguments are constructed. Else, it becomes a double addition(not that it is an issue) which is superfluous.
 
 I would suggest to go with just arm_nnfunctions.h as the include file here.

Reply:
Good to know. I'll let you guys decide since it is your API. If you'd like to go with only arm_nnfunctions.h (with the full path), that is fine with me.